Design-Process.md

```diff
@@ -0,0 +1,194 @@
+
+#### Lifecycle
+
+* Starts when an [implemented feature](#implemented-feature) is shipped as part of a C# release
+* This is the final state for a feature that is included in C#, no further state changes occur
+
+### Rejected feature
+
+When a feature is explicitly considered during an LDM, and the LDT decides as a group to reject it, it moves to this state. At this point, close the champion issue as not planned
+and set the milestone to `Likely Never`. It's not impossible for an issue to be pulled back out of this state and included in the language in the future, but generally, this state
+means that the feature will never be part of C#.
+
+#### Lifecycle
+
+* Starts when a [championed feature](#championed-feature) is considered at LDM and rejected
+* While it is possible that some rejected features end up getting reconsidered, this is generally the final state for language features that are explicitly considered and
+  rejected during LDM
+
+## Language Design Team processes
+
+These are various processes and actions taken by LDT members during the development of a feature. Community members should not perform these actions unless invited to do so
+by an LDT member.
+
+### Steps to move a [Discussion](#proposed-feature) to a [Champion feature](#championed-feature)
+
+When an LDT member decides to champion a discussion, they take the following steps:
+
+1. Create a new proposal champion issue.
+    * If preferred, the LDT member can ask the original proposer to create this issue.
+    * Note: it can be easier to create the PR for step 6 first, get that into a ready-to-merge state, and then create the champion issue at that point, depending on the complexity
+      of the feature.
+2. Assign themselves to the champion issue.
+3. Apply the `Proposal Champion` label to the new issue, as well as to the original discussion.
+4. Link to the original discussion from the champion issue.
+5. Lock the proposal champion issue for comments to ensure that discussion continues in the discussion area, rather than on the champion issue.
+6. Fill out and check in a [proposal template](proposals/proposal-template.md) for the feature. Exact specese is not required, but there should be enough detail to have a
+   meaningful triage session.
+    * This is also something the LDT member can ask a community member to open a PR for, if they are willing.
+    * The filled out proposal should include a link to the champion issue for easy navigation.
+
+### Bringing open questions to LDM
+
+During the course of development of a feature, there are several different types of questions that need to get brought to LDM for answers. The most important overriding factor
+for any question is that there is a checked-in commit that contains the question. The document and commit will be linked as part of the notes so that future readers of the notes
+can understand the full context in which the question was asked.
+
+#### Alternative proposals, supplemental documentation
```

Comment:
@CyrusNajmabadi that's this section; is there some other wording I can use here to make it clearer to you?